### ISSUE TITLE
Workflow and consistency

### ISSUE THREAD

**Issue by Sh0ri** [Gacara/shasser#1]
### Summary
Follow a simplified workflow to learn how to work with a team, and improve code consistency. **Obviously** the following recommendations are **overkill** for this project, buts its nice to have them somewhere in your head and to try to follow the a bit. The key parts are **pull requests** and **code consistency**  

### Workflow

Future improvements to the project should follow this schema :
1. **Design.**
Create an issue to determine what would be the best design for the functionality
2. **API.**
Create all issues related to the API modifications for the functionality
3. **Web.**
Create all issues related to the web modifications for the functionality

### Branches have to start by one of the following terms:

- `release` followed by the name of the release and a name for the feature you're working on (for example `release/pokemon-creation/navigation`)
- `fix` when it’s **a fix** on something that has already been done and shipped (for example `fix/pokemon-listing`)
- `update` when it’s either **an update** to a newer library or an update of an old piece of code (for example `update/authentification`)

### Pull requests
Once the work is done, push your code, create a new draft pull request, assign yourself, and review your own code. A good pull request has to be very specific:

- Explain the context: why this work was needed.
- What has been done: list all the changes you submitted to the codebase.
- What’s related:
    - Always link the pull request with the related issues and other pull requests (using the `Fixes` or `Closes` keywords)
    - If a migration script needs to be run when the pull request is merged, specify it.
    - If the pull request depends on another pull request, specify it too.
- What comes next: how the current push can be improved in the future.

> It’s important to keep pull requests small enough to be reviewed. From what we experienced, a good pull request is under 12 files and under 500 lines.

Once you reviewed your own pull request, assign a reviewer. Don't forget that pull requests have to be assigned at all time until they are closed.

### Code consistency
Add [eslint](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb) package to visual code, and use the airbnb configuration
